tests/e2e/e2e_retry_policy_test.go

```diff
@@ -125,6 +125,7 @@ var _ = OSMDescribe("Test Retry Policy",
 
 					By("A request that will be retried NumRetries times then fail")
 					err = wait.Poll(time.Second*3, time.Second*30, func() (bool, error) {
+						defer GinkgoRecover()
```

Comment:
For my own understanding, should `GinkgoRecover()` be added to all `wait.Poll`s?

Reply:
Yeah I'm just playing around at this point. My dev machine is crawling, so I'm pushing to a PR branch for (more) rapid iteration

Reply:
Got it!!